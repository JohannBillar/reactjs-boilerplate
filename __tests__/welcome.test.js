import React from 'react';
import renderer from 'react-test-renderer';
import Welcome from '../app/components/Welcome';

describe('Welcome (Snapshot)', () => {
  it('Welcome renders Hello World', () => {
    const component = renderer.create(<Welcome />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
