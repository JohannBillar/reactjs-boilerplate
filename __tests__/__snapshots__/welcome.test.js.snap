exports[`Welcome (Snapshot) Welcome renders Hello World 1`] = `
<div>
  Hello World
</div>
`;
